app/scripts/modules/core/src/serverGroup/details/ShowUserData.tsx

```diff
@@ -45,11 +53,20 @@ export class ShowUserData extends React.Component<IShowUserDataProps, IShowUserD
             </h3>
           </Modal.Header>
           <Modal.Body>
-            <div className="modal-body">
-              <textarea readOnly={true} rows={15} className="code">
-                {userData}
-              </textarea>
-            </div>
+            <>
+              <textarea
+                className="code"
+                readOnly={true}
+                rows={15}
+                value={decodeAsText ? decodeUnicodeBase64(userData) : userData}
```

Comment:
this moved to `value` because you're not actually supposed to use `children` with `textarea` elements in React (it essentially opts you out of React managing the text and means you're responsible for keeping it in sync).